src/ProjectService.Business/Commands/Project/EditProjectCommand.cs

```diff
@@ -93,21 +96,23 @@ public OperationResultResponse<bool> Execute(Guid projectId, JsonPatchDocument<E
 
             GetProjectFilter filter = new GetProjectFilter { ProjectId = projectId };
 
-            DbProject dbProject = _repository.Get(filter);
+            DbProject dbProject = _projectRepository.Get(filter);
+            IEnumerable<DbProjectUser> dbUsers = _userRepository.GetProjectUsers(projectId, false);
 
             OperationResultResponse<bool> response = new();
+            Guid id = _httpContextAccessor.HttpContext.GetUserId();
```

Comment:
rename to more understandable variable, which id?

Reply:
done